src/js/components/Card.js

```diff
@@ -34,13 +34,14 @@ export default class Card extends Component {
   }
 
   _renderLink () {
-    const { link } = this.props;
+    const { link, linkIcon, linkText } = this.props;
 
     if (link) {
       return (
-        <Box pad={{vertical: "small"}}>
-          {link}
-        </Box>
+        <Anchor className={`${CLASS_ROOT}__link`} primary={true} href={link}
```

Comment:
I think what sort of component this is will need to be under the caller's control. For instance, when using react-router, this will want to be a Link, or something like NavAnchor in phoenix or grommet-docs.